go.sum

```diff
@@ -63,6 +63,10 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
 github.com/ChainSafe/chaindb v0.1.5-0.20210608140454-9606fe8c3985 h1:jyFsOjzTMoRwNvmW/OORZpffmItkoLvsbxB8koHX4ns=
 github.com/ChainSafe/chaindb v0.1.5-0.20210608140454-9606fe8c3985/go.mod h1:P01m9E6xj6Mps1rtf7SurEX9oOcy1jYEyccZQAEw9+4=
+github.com/ChainSafe/chaindb v0.1.5-0.20220321221626-9cc45c8bdd62 h1:5pTK7REpz8B9ioDp4XcOzYfbaMa2S91/DLWsCfvNmpY=
```

Comment:
run `go mod tidy`. On my machine this gets cleaned up.

Reply:
done.